components/context-dialog/README.md

```diff
@@ -89,6 +89,8 @@ The context actions menu is a variant of the context dialog placed on the top
 right within a [card](/components/card), containing
 [secondary buttons](/components/button).
 
+<ba-live-example name="DtExampleContextDialogActions" background="true"></ba-live-example>
```

Comment:
you can just set the background attribute without the `true` value (in all files, also for the fullwidth attr)